unzip/README.md

```diff
@@ -0,0 +1,24 @@
+# Build unzip as static PIE
+
+Build the [unzip](https://infozip.sourceforge.net/UnZip.html) as static PIE ELF running on Linux
+
+## Requirements
+
+Make sure the following packages are installed:
+
+* GCC
+* GNU Make
+
+## Build
+
+The `unzip` static PIE ELF file is located in the current directory.
+In order to rebuild it you have to run the build.sh bash script. The 
```

Comment:
Please use one sentence per line and one line per sentence.

Reply:
Changes done